libs/markets/src/lib/__generated__/markets.ts

```diff
@@ -104,7 +104,6 @@ export const MarketFieldsFragmentDoc = gql`
     open
     close
   }
-  successorMarketID
```

Comment:
this will cause test failing in apps/trading/client-pages/markets/closed.spec.tsx

Reply:
Yup I noticed. Fixed now :-)